agent_build_refactored/managed_packages/scalyr_agent_python3/agent-python3-config

```diff
@@ -0,0 +1,193 @@
+#!/opt/scalyr-agent-2-dependencies/bin/python3
```

Comment:
Just to confirm, we also run all the lint checks on the ``agent_build_refactored/`` directory, right?

Reply:
strange, it is not added to tox, but  it still was checked by the lint job. 
Anyway, added it to tox.